imagetagger/imagetagger/images/views.py

```diff
@@ -86,12 +87,36 @@ def index(request):
     ).select_related('team').filter(team__in=userteams).order_by('-priority', '-time')
     imageset_creation_form = ImageSetCreationFormWT()  # the user provides the team manually
     imageset_creation_form.fields['team'].queryset = userteams
+    all_images = Image.objects.all().select_related('image_set').distinct()
+    public_images = all_images.filter(image_set__public=True)
+    annotation_types = AnnotationType.objects.all()
+    all_imagesets = ImageSet.objects.all()
+    all_annotations = Annotation.objects.all()
+    public_annotations = all_annotations.filter(image__image_set__public=True)
+    annotation_counts = [(annotation_type.name, all_annotations.filter(annotation_type=annotation_type).count(), public_annotations.filter(annotation_type=annotation_type).count()) for annotation_type in annotation_types]
+    annotation_counts.sort(key=lambda a: a[1], reverse=True)
+    all_users = User.objects.all()
+    active_users = all_users.filter(points__gte=50)
+    all_teams = Team.objects.all()
+    active_teams = Team.objects.annotate(mc=Count('members', filter=Q(memberships__user__in=User.objects.annotate(annotation_count=Count('creator')).filter(points__gte=50)))).filter(mc__gte=2)
```

Comment:
Why would you annotate the annotation count for the user? The result does not seem to be used at all, and the annotation count could be quite expensive with a lot of rows in the annotation table.